detection_rules/main.py

```diff
@@ -149,12 +151,48 @@ def view_rule(ctx, rule_id, rule_file, api_format):
     if not rule:
         client_error('Unknown format!')
 
-    click.echo(toml_write(rule.rule_format()) if not api_format else
-               json.dumps(rule.contents, indent=2, sort_keys=True))
+    if verbose:
+        click.echo(toml_write(rule.rule_format()) if not api_format else
+                   json.dumps(rule.contents, indent=2, sort_keys=True))
 
     return rule
 
 
+@root.command('export-rules')
+@click.argument('rule-id', nargs=-1, required=False)
+@click.option('--rule-file', '-f', multiple=True, type=click.Path(dir_okay=False), help='Export specified rule files')
+@click.option('--directory', '-d', multiple=True, type=click.Path(file_okay=False),
+              help='Recursively export rules from a directory')
+@click.option('--outfile', '-o', default=get_path('exports', f'{time.strftime("%Y%m%dT%H%M%SL")}.ndjson'),
+              type=click.Path(dir_okay=False), help='Name of file for exported rules')
+@click.option('--randomize-id', '-r', is_flag=True, help='Randomize rule IDs before export')
```

Comment:
"randomize" doesn't seem quite like the best word here. i'm not sure what an alternative is, but maybe a `--replace-id` or something. could go the opposite and do `--preserve-id` like #613 but it all depends on what we want the default behavior to be.

if you do change, don't forget to update the docs

Reply:
I can work with `replace-id` 👍